master/internal/task/allocation.go

```diff
@@ -399,6 +400,7 @@ func (a *allocation) WatchRendezvous(rID sproto.ResourcesID) (RendezvousWatcher,
 
 	if a.rendezvous == nil {
 		a.rendezvous = newRendezvous(a.model.AllocationID, a.resources, rendezvousTimeoutDuration)
+		a.closers = append(a.closers, a.rendezvous.close)
```

Comment:
what happens with this if we restore the allocation during its rendezvous?

The other calls to closers happens in resourcesAllocated so it seems pretty clear it works across restore.

(I'm not saying this is incorrect just not obvious to me how this interacts with restore)